packages/badge/src/Default/styles.module.scss

```diff
@@ -1,31 +1,14 @@
-@use '@owlui/theme/src' as theme;
```

Comment:
updated to use new utils from theme `https://osgca.slack.com/archives/C02NQ6T65T6/p1656601104477139`